src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1051,6 +1062,56 @@ private void RegisterHomeWorkspace(HomeWorkspaceModel newWorkspace)
 
         #endregion
 
+        #region backup/timer
+
+        /// <summary>
+        /// Backup all the files
+        /// </summary>
+        protected void SaveBackupFiles(object state)
+        {
+            DynamoModel.OnRequestDispatcherBeginInvoke(() =>
+            {
+                foreach (var workspace in Workspaces)
+                {
+                    if (!workspace.HasUnsavedChanges)
+                        continue;
+
+                    string fileName;
+                    if (string.IsNullOrEmpty(workspace.FileName))
+                    {
+                        fileName = Configurations.BackupFileNamePrefix + workspace.Guid;
+                        var ext = workspace is HomeWorkspaceModel ? ".DYN" : ".DYF";
+                        fileName += ext;
+                    }
+                    else
+                    {
+                        fileName = Path.GetFileName(workspace.FileName);
+                    }
+
+                    var savePath = Path.Combine(pathManager.BackupDirectory, fileName);
+                    workspace.SaveAs(savePath, null);
+                    Logger.Log("Backup file is saved: " + savePath);
```

Comment:
If we're sure `Logger.Log` doesn't end up sending the string to the Console window, then this doesn't need localization.